ui/.storybook/preview-head.html

```diff
@@ -0,0 +1,2 @@
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet"> 
```

Comment:
Can we change this to static fonts? There are some concerns about GDPR and Google Fonts.

https://complianz.io/google-fonts-and-gdpr-does-it-work/